openshift/release/kn.yaml

```diff
@@ -3,8 +3,8 @@ plugins:
     module: knative.dev/kn-plugin-source-kafka
     pluginImportPath: knative.dev/kn-plugin-source-kafka/plugin
     version: v0.25.0
-#  - name: func
-#    module: knative.dev/kn-plugin-func
-#    pluginImportPath: knative.dev/kn-plugin-func/plugin
-#    version: v0.18.0
+  - name: func
+    module: knative.dev/kn-plugin-func
+    pluginImportPath: knative.dev/kn-plugin-func/plugin
+    version: main
```

Comment:
This version wasn't updated, but it's used only in our helper tool to add plugins. The correct values are updated in go.mod.